php-checks/src/main/java/org/sonar/php/checks/wordpress/WordPressFileEditorCheck.java

```diff
@@ -50,9 +50,6 @@ public void visitCompilationUnit(CompilationUnitTree tree) {
           .ifPresent(value -> newIssue(fileEditConfigTree, MESSAGE));
       }
     }
-
-    fileEditConfigTree = null;
-    fileModsConfigTree = null;
```

Comment:
When I took a look for the second time, I could not see the reason for it.

Reply:
Clearing references to trees after the check is run. I vaguely remember we had a problem because of that once. Probably here it would make no difference (it is only references to small function calls), but it does hurt either.